integration-tests/e2e/emaildomain.create.cy.js

```diff
@@ -0,0 +1,81 @@
+const MenuPage = require('../pages/menuPage')
+const EmailDomainListingPage = require('../pages/emailDomainListingPage')
+const EmailDomainCreationPage = require('../pages/emailDomainCreationPage')
+
+context('CreateEmailDomain', () => {
+  before(() => {
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('Should create new email domain, after completing form and clicking the  create email domain button', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_EMAIL_DOMAINS' }] })
+    cy.signIn()
+    cy.task('stubAllEmailDomains', {})
+    MenuPage.verifyOnPage().viewEmailDomainListing()
+    const emailDomainListingPage = EmailDomainListingPage.verifyOnPage()
+    emailDomainListingPage.navigateToCreateEmailDomainPageButton().click()
+    const emailDomainCreationPage = EmailDomainCreationPage.verifyOnPage()
+    emailDomainCreationPage.createEmailDomainPage('Domain1', 'Domain1Description')
+    cy.task('verifyGetAllEmailDomains').should((requests) => {
+      expect(requests).to.have.lengthOf(1)
```

Comment:
Not sure what this test is doing - the verifyGetAllEmailDomains just verifies that the method has been called

Reply:
Yes, it does just that. The create email  domain calls the email domain listing, so its a good check.

Reply:
I don't think this is still quite right.  We are still only checking that the getAllDomainsList has been called - not what the test specifies.

Reply:
I don't think this is still quite right. We are still only checking that the getAllDomainsList has been called - not what the test specifies. If you're doing the test using the verify - it should verifyAddEmailDomain method is called. Are you stubbing this method out?